### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by MaxGraey**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,11 +12,11 @@
   },
   "dependencies": {
     "@protobufjs/utf8": "^1.1.0",
-    "binaryen": "46.0.0-nightly.20180427",
+    "binaryen": "47.0.0-nightly.20180503",
     "glob": "^7.1.2",
     "long": "^4.0.0",
     "minimist": "^1.2.0",
-    "ts-node": "^5.0.1"
+    "ts-node": "^6.0.2"
```
Comment (dcodeIO) [inline package.json]: I had some issues with v6 when running the sources. Wasn't finding some things. Doesn't this happen for you?

Comment (MaxGraey) [reply package.json]: Hmm, In Mac OSX it seems everting ok with tests. How this checked?

Comment (dcodeIO) [reply package.json]: Iirc `npm run clean` `npm run test` `npm run build`, in this order, somehow triggered it.

Comment (MaxGraey) [reply package.json]: Are you sure it should be `I64` instead `i64`?

Comment (dcodeIO) [reply package.json]: Yeah, something like that. Ideally, we should fix the underlying issue, of course, so we can use v6 :)

Comment (MaxGraey) [reply package.json]: Ok, I fixed this =)

Comment (MaxGraey) [reply package.json]: Forgot about non-distribution requirements

**Comment by dcodeIO** [comment]
Thanks :)

**Pull request merged**